portality/migrate/3245_ur_vs_app/operations.py

```diff
@@ -0,0 +1,7 @@
+from portality import constants
+def make_update_request(application):
+    if application.related_journal:
```

Comment:
I'd add a `and not application.application_type == constants.APPLICATION_TYPE_UPDATE_REQUEST` check here too, just to make sure we're not calling save on applications that don't need it